### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by pborreli**

### CODE REVIEW

File: docs/building/osx.md [hunk 1]
```diff
@@ -78,4 +78,4 @@ start a `powershell` session, and use `Start-PSBuild` from the module.
 The output directory will be slightly different because your runtime identifier is different.
 PowerShell will be at `./src/powershell-unix/bin/Linux/netcoreapp1.0/osx.10.11-x64/powershell`,
 or `osx.10.10` depending on your operating system version.
-Note that configration is still `Linux` because it would be silly to make yet another separate configuration when it's used soley to work-around a CLI issue.
+Note that configuration is still `Linux` because it would be silly to make yet another separate configuration when it's used soley to work-around a CLI issue.
```
Comment (alexandair) [inline docs/building/osx.md]: Please, fix "soley" -> "solely" while you are at it. :)

**Comment by TravisEz13** [comment]
I agree with fixing the other typos, but this is good.  File an issue or submit another PR.

**Comment by TravisEz13** [comment]
@alexandair , thanks for reviewing.  
@pborreli , thanks for the contribution

**Comment by alexandair** [comment]
@TravisEz13 Just opened a new PR and found another typo in osx.md.

**Pull request merged**